lib/compat/wordpress-6.5/block-bindings/sources/pattern.php

```diff
@@ -4,8 +4,8 @@
  *
  * @package gutenberg
  */
-if ( function_exists( 'wp_block_bindings_register_source' ) ) {
-	$pattern_source_callback = function ( $source_attrs, $block_instance, $attribute_name ) {
+if ( ! function_exists( 'gutenberg_register_block_bindings_pattern_overrides_source' ) && ! function_exists( 'gutenberg_block_bindings_pattern_overrides_callback' ) ) {
```

Comment:
You can remove these function checks. They are only necessary for functions that we expect to be defined outside the Gutenberg plugin. In our case, it's only for functions and classes defined in WordPress Core.